docs/buildtest_perlmutter.rst

```diff
@@ -36,32 +40,74 @@ In this exercise, we will submit a batch job that will run `hostname` in the slu
+    hostname_perlmutter/b1d7b318: JobID 4990983 dispatched to scheduler
+    Polling Jobs in 30 seconds
+    hostname_perlmutter/80e317c1: Job 4990982 is complete!
+    hostname_perlmutter/80e317c1: Test completed in 30.248888 seconds
+    hostname_perlmutter/80e317c1: Test completed with returncode: 0
+    hostname_perlmutter/80e317c1: Writing output file -  /global/u1/s/siddiq90/gitrepos/buildtest/var/tests/perlmutter.slurm.debug/solution/hostname_perlmutter/80e317c1/hostname_perlmutter.out
+    hostname_perlmutter/80e317c1: Writing error file - /global/u1/s/siddiq90/gitrepos/buildtest/var/tests/perlmutter.slurm.debug/solution/hostname_perlmutter/80e317c1/hostname_perlmutter.err
+                                    Pending and Suspended Jobs
+    ┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━┳━━━━━━━━━┓
+    ┃ builder                      ┃ executor                 ┃ jobid   ┃ jobstate ┃ runtime ┃
+    ┡━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━╇━━━━━━━━━┩
+    │ hostname_perlmutter/b1d7b318 │ perlmutter.slurm.regular │ 4990983 │ PENDING  │ 30.455  │
+    └──────────────────────────────┴──────────────────────────┴─────────┴──────────┴─────────┘
+                                          Completed Jobs
+    ┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━━┳━━━━━━━━━━━┓
+    ┃ builder                      ┃ executor               ┃ jobid   ┃ jobstate  ┃ runtime   ┃
+    ┡━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━━╇━━━━━━━━━━━┩
+    │ hostname_perlmutter/80e317c1 │ perlmutter.slurm.debug │ 4990982 │ COMPLETED │ 30.248888 │
+    └──────────────────────────────┴────────────────────────┴─────────┴───────────┴───────────┘
+
 
 Exercise 2: Performing Status Check
 ------------------------------------
 
-In this exercise, we will check version of Lmod via environment **LMOD_VERSION** and specify the
-the output using :ref:`regular expression <regex>`.
+In this exercise, we will check the version of Lmod using the environment variable **LMOD_VERSION** and specifying the
+the output using a :ref:`regular expression <regex>`. We will run the test with an invalid regular expression and see if test fails and
+rerun example until it passes
 
 .. literalinclude:: ../perlmutter_tutorial/ex2/module_version.yml
    :language: yaml
 
-This buildspec is invalid, your first task is to make sure buildspec is valid. Once you have accomplished this task, try building
-the test and check the output of test. If your test passes, try updating the regular expression and see if test fails. Revert the change
-back and make the test pass.
+First let's try running this test, you will notice the test will fail validation::
+
+    buildtest build -b perlmutter_tutorial/ex2/module_version.yml
+
+
+**TODO:**
+
+- Validate the buildspec using ``buildtest buildspec validate``
+- Add a regular expression on ``stdout`` stream and make sure test fails
```

Comment:
Suggestion:

```
Add a regular expression to stdout and make sure the test fails
```

I would also re-evaluate what stdout is here. You don't use the word stream in this document that I've seen so why here?

Reply:
yeah this was a hint since you have to do `stream: stdout` see https://buildtest.readthedocs.io/en/devel/buildspecs/buildspec_overview.html#passing-test-based-on-regular-expression